### PR TITLE
Preparing release 1.17.1

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
 
-1. Make sure you are using an operator-sdk newer than 0.10.0
-
-1. Make sure the `versions.txt` file is up to date
+1. Update Jaeger version in `versions.txt`
 
 1. Make sure the new version is present at `pkg/upgrade/versions.go`
 
-1. Prepare a changelog and get it merged. A list of commits since the last release (`v1.15.1` in the following example) can be obtained via:
+1. Prepare a changelog since last release:
```
Comment (jpkrohling) [inline RELEASE.md]: From the original PR:

```md
last release, with an [OAuth Token](https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token) with the scopes X, Y and Z:
```

**Comment by pavolloffay** [comment]
> LGTM, just needs to tell how the OAuth Token should be created.

That is provided in the `changelog` target

**Comment by jpkrohling** [comment]
```
That is provided in the changelog target
```

The instructions on how to properly run the `changelog` is being provided after the `changelog` is run. It should really be part of the RELEASE.md, and people should be expected to create the OAuth Token before running the target ;-) 

Besides, the comment is missing which scopes should be selected...

**Comment by pavolloffay** [comment]
PR updated and added instructions on how to generate the oauth token.

The `examples1` pass locally on minikube:

```
NAMESPACE=pavolloffay make e2e-tests-examples1                                    10:56 
Formatting code...
Building...
Sending build context to Docker daemon    174MB
Step 1/7 : FROM registry.access.redhat.com/ubi8/ubi
 ---> 0c46e5c7a82a
Step 2/7 : ENV OPERATOR=/usr/local/bin/jaeger-operator     USER_UID=1001     USER_NAME=jaeger-operator
 ---> Using cache
 ---> d443d14f2a57
Step 3/7 : RUN INSTALL_PKGS="       openssl       " &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all &&     mkdir /tmp/_working_dir &&     chmod og+w /tmp/_working_dir
 ---> Using cache
 ---> df5ea8178a31
Step 4/7 : COPY scripts/* /scripts/
 ---> Using cache
 ---> bb623e0ba6fd
Step 5/7 : COPY build/_output/bin/jaeger-operator ${OPERATOR}
 ---> b4b069c6d67b
Step 6/7 : ENTRYPOINT ["/usr/local/bin/jaeger-operator"]
 ---> Running in 070f8b3ccb10
Removing intermediate container 070f8b3ccb10
 ---> a3d61dad3422
Step 7/7 : USER ${USER_UID}
 ---> Running in c103c146e772
Removing intermediate container c103c146e772
 ---> 0f8df52b67c4
Successfully built 0f8df52b67c4
Successfully tagged pavolloffay/jaeger-operator:latest
Pushing image pavolloffay/jaeger-operator:latest...
Creating namespace default
# Elasticsearch requires labeled nodes. These labels are by default present in OCP 4.2
node/minikube not labeled
# This is not required in OCP 4.1. The node tuning operator configures the property automatically
# when label tuned.openshift.io/elasticsearch=true label is present on the ES pod. The label
# is configured by ES operator.
customresourcedefinition.apiextensions.k8s.io/prometheusrules.monitoring.coreos.com configured
customresourcedefinition.apiextensions.k8s.io/servicemonitors.monitoring.coreos.com configured
serviceaccount/elasticsearch-operator unchanged
clusterrole.rbac.authorization.k8s.io/elasticsearch-operator configured
clusterrolebinding.rbac.authorization.k8s.io/elasticsearch-operator-rolebinding configured
customresourcedefinition.apiextensions.k8s.io/elasticsearches.logging.openshift.io configured
deployment.apps/elasticsearch-operator configured
deployment.apps/elasticsearch-operator image updated
Running Example end-to-end tests part 1...
ok  	github.com/jaegertracing/jaeger-operator/test/e2e	107.818s

```

**Comment by pavolloffay** [comment]
PR https://github.com/jaegertracing/jaeger-operator/pull/973 shows that `examples1` fail also on `master` branch.

**Pull request merged**